### ISSUE TITLE
pkvldtprod needs a valid SSL certificate (Error 13275)

### ISSUE THREAD

**Issue by InternalLoss** [mm201/pkmn-classic-framework#58]
Since the new DS flaw regarding SSL was found, many have been trying to use Wiimmfi and other alt services again (because now no patch for SSL is needed). However, I've been told that while Gen 4's pkmn-classic works fine, Gen 5's is non-functional:

"Gen 5's GTS, you can connect, you can search, but you can't trade"

would this be able to be looked at?

**Comment by mm201** [comment]
Yeah I know exactly what the problem is. pkvldprod needs a certificate. I’ll get to it soon. In the meantime, you can still connect with a nossl patch.

**Comment by InternalLoss** [comment]
Got it, awesome; thanks for the info.

As long as it supports SSLv3, should be ok c:

**Comment by InternalLoss** [comment]
After a lot of walking through issues, I was able to fix gen 4 on my side (i forgot ProxyPreserveHost xP) - as WFC Plaza now works on Wiimmfi, the only issue we have left is Trainer Rankings on Gen 4's side :D 

Some people were wondering about the ETA of a certificate on the gen 5 servers, since gen 5 is now a game you can connect to without any WEP internet - just drop in the dns and off you go :p 

Another question; is there any other way to contact you than GitHub issues? xP

**Comment by mm201** [comment]
So apparently nds-constraint doesn't work with the pkvldtprod server in its current state. I'm asking shutterbug to look into it. Certainly not a quick fix.

**Comment by Blayer98** [comment]
Hi, any updates on this? Would love to be able to battle and trade without having to resort to friend codes. :)

**Comment by InternalLoss** [comment]
There isn't something we can just go :sparkles:POOF:sparkles: it's fixed; if  nds-constraint's cert cannot validly sign a certificate for pkvldtprod, there's no way to make it work without NoSSL of some kind.

**Comment by Blayer98** [comment]
Ok, I've tried using a NoSSL patched rom, but I'm still getting communication errors and the 13275 errors.

I looked at packet logs and the standard query response is showing the DNS as 111.168.21.99 which is pkvldtprod.nintendo.co.jp...
this is when uploading battle videos. receiving battle videos properly redirects to pkmnclassic.net.
the DNS error also happens with Random Matchup and the GTS also has the wrong DNS being assigned to pkvldtprod.nintendo.co.jp.
